File: ms-frontend/types/Catalogo.ts
import { FormItems } from "./FormTypes";
import Marca from "./Marca";
import TipoEquipo from "./TipoEquipo";
import Uso from "./Usos";

type Catalogo = {
  marcas: Marca[];
  tipos: TipoEquipo[];
  usos: Uso[];
}

export default Catalogo;

File: ms-equipos/src/main/java/com/usach/msequipos/dtos/Catalogo.java
package com.usach.msequipos.dtos;

import com.usach.msequipos.entities.Marca;
import com.usach.msequipos.entities.TipoEquipo;
import com.usach.msequipos.entities.Uso;
import lombok.*;

import java.util.List;

@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Catalogo {
    public List<Marca> marcas;
    public List<TipoEquipo> tipos;
    public List<Uso> usos;
}
